testnets/junotestnet/chain.json

```diff
@@ -5,7 +5,7 @@
   "network_type": "testnet",
   "pretty_name": "Juno Testnet",
   "chain_type": "cosmos",
-  "chain_id": "uni-6",
+  "chain_id": "uni-7",
```

Comment:
It looks like there are existing registered IBC connections which describe uni-6 with other chains, so we should create an archive chain for uni-6, (such as 'junotestnet6') and update its ibc files accordingly.

Reply:
ok! i'll do it